filament/include/filament/IndexBuffer.h

```diff
@@ -139,7 +139,7 @@ class UTILS_PUBLIC IndexBuffer : public FilamentAPI {
      * @param engine Reference to the filament::Engine to associate this IndexBuffer with.
      * @param externalBuffer Pointer to the external buffer that will be used in this buffer slot.
```

Comment:
```suggestion
     * @param externalBuffer Pointer to the external buffer that will be used.
```
The comment is bad copy-paste from VertexBuffer.h. For index buffers you don't have multiple slots (unlike for vertex buffers)